lib/blueprinter/view_collection.rb

```diff
@@ -35,7 +35,16 @@ def fields_for(view_name)
     end
 
     def transformers(view_name)
-      views[view_name].transformers
+      transformers = gather_transformers_from_included_views(view_name)
+      transformers.presence || views[:default].transformers
+    end
+
+    def gather_transformers_from_included_views(view_name)
+      current_view = views[view_name]
+      current_view.included_view_names.each_with_object([]) do |included_view_name, transformers|
+        next if view_name == included_view_name
+        transformers.concat(gather_transformers_from_included_views(included_view_name))
+      end.concat(current_view.view_transformers)
```

Comment:
I think we need to ensure that transformers are run "at-most" once. we can do that by using `.uniq` or by using a `Set` instead of `Array` in `each_with_object`. Does this also solve the inheritance [problem](https://github.com/procore-oss/blueprinter/issues/225)? IMO, we need to ensure transformers get inherited from base class as well.

Reply:
Sure @ritikesh. Will add Uniq. And yes it will take the included views transformers in the included order. 
If no transformers are present, it will take the default views transformer or else global transformer